MSMetaEnhancer/libs/services/CIR.py

```diff
@@ -25,7 +25,7 @@ async def casno_to_smiles(self, cas_number):
         args = f'{cas_number}/smiles?resolver=cas_number'
         response = await self.query_the_service('CIR', args)
         if response:
-            return {'smiles': response}
+            return {'smiles': response.split('\n')[0]}
```

Comment:
If this function retrieves the first element out of a list or so, it could be actually refactored into one that has a descriptive name.